### PR TITLE
use flake8 defaults

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: redshift_sqlalchemy/dialect.py [hunk 1]
```diff
@@ -484,7 +487,8 @@ def get_unique_constraints(self, connection, table_name,
         """
         Return information about unique constraints in `table_name`.
 
-        See :meth:`~sqlalchemy.engine.interfaces.Dialect.get_unique_constraints`.
+        See
+        :meth:`~sqlalchemy.engine.interfaces.Dialect.get_unique_constraints`.
```
Comment (graingert) [inline redshift_sqlalchemy/dialect.py]: I don't think we can cross reference like this. I think a better approach is to link to the docs using HYPERTEXT

Comment (jklukas) [inline redshift_sqlalchemy/dialect.py]: If we build docs using sphinx, we can add sqlalchemy to the intersphinx mappings, which enables links like these. I created the basic structure for this in [a new sphinx branch of this repo](https://github.com/graingert/redshift_sqlalchemy/tree/sphinx). I was able to build those docs and test that these links work.

Comment (graingert) [inline redshift_sqlalchemy/dialect.py]: Ah nice

Comment (graingert) [inline redshift_sqlalchemy/dialect.py]: Does it work with this line split here?

Comment (jklukas) [inline redshift_sqlalchemy/dialect.py]: The line split is not a problem.

**Comment by jklukas** [comment]
+1

**Pull request merged**